tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -163,7 +163,7 @@ public function getSpaceIdByNameFromResponse(string $name): string {
 	 * @throws Exception
 	 */
 	public function getSpaceByNameFromResponse(string $name): array {
-		$response = json_decode($this->featureContext->getResponse()->getBody(), true, 512, JSON_THROW_ON_ERROR);
+		$response = json_decode((string)$this->featureContext->getResponse()->getBody(), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
```suggestion
		$response = json_decode($this->featureContext->getResponse()->getBody()->getContents(), true, 512, JSON_THROW_ON_ERROR);
```

Can we do this to always get a string?

Reply:
I tried doing this but the tests failed.
https://drone.owncloud.com/owncloud/ocis/8104/20/6